broadinstitute/single_cell_portal_core
# Add "Request a feature" link and Zendesk form (SCP-5567)

This adds a clean, clear, and more structured way for users to ask for an enhancement, indicate if they'd sponsor it, etc.

Here's how it looks!

https://github.com/broadinstitute/single_cell_portal_core/assets/1334561/9e6609bc-9aed-4efa-aadd-1ce39d09dd1a

I tried adding a new type of Zendesk form, but it was inaccessible to respondents and our support account.  That could probably be debugged with more work, but this simpler approach of adapting the general form seems likely sufficient for our needs.

_Reviewer note: to ease any manual tests, I branched this work from the other open PR, and merged `development` into this branch.  The only code change for this PR is bd55ef7._

### Test
To manually test:
* Go to SCP
* Click "Help" at top right
* Click "Request a feature"
* Confirm Zendesk form loads with content as shown in video above.

This satisfies SCP-5567.
